CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.2] - 2023-10-05
+
+### Fixed
+- Removed 'Hartgroepen 1 t/m 5' from pilot
```

Comment:
Nog toevoegen dat dit van revalidatie is? Zou anders denkden dat dit een cardiogroep is

Reply:
Misschien sowieso goed om bij al die exclusies aan te geven bij welke poli het hoort, of is dat evident?